package/CHANGELOG

```diff
@@ -13,28 +13,30 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 kain88-de, jbarnoud
+??/??/16 tyler.je.reddy, kain88-de, jbarnoud, richardjgowers
 
-  * 0.13.1
+  * 0.14.0
```

Comment:
Yeah sorry, that was me.  I don't think we'll have time for a 13.1, so everything is becoming 14